src/sms_inbound.js

```diff
@@ -21,7 +21,7 @@ go.app = function() {
                 case "BABY": case "NWA":
                     return self.states.create("state_change_baby");
                 default:
-                    return self.states.create("state_save_inbound");
+                  // return self.states.create("state_save_inbound");
```

Comment:
We can probably remove this completely. Comments just creates unnecessary clutter in the code and Github will keep the old code for us if we ever want to revert.

Also look at removing the `state_save_inbound` state completely if nothing else is using it.